Spigot-Server-Patches/0179-Allow-Reloading-of-Command-Aliases.patch

```diff
@@ -0,0 +1,29 @@
+From 094172b70ec10ecd48d1093f0425f88afcce397e Mon Sep 17 00:00:00 2001
+From: willies952002 <admin@domnian.com>
+Date: Mon, 21 Nov 2016 23:11:25 -0500
+Subject: [PATCH] Allow Reloading of Command Aliases
+
+Reload the aliases stored in commands.yml
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+index d5e602f..f429562 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+@@ -1861,5 +1861,14 @@ public final class CraftServer implements Server {
+         }
+         return entity.getBukkitEntity();
+     }
++
++    @Override
++    public void reloadCommandAliases() {
++        commandsConfiguration = YamlConfiguration.loadConfiguration(getCommandsConfigFile());
++        getCommandAliases().keySet().forEach(alias -> {
++            getCommandMap().getKnownCommands().remove(alias.toLowerCase(java.util.Locale.ENGLISH));
```

Comment:
This whole block can be replaced with 
Set<String> keys = getCommandMap().getKnownCommands().keySet();

then keys.removeAll(getCommandAliases().keySet());

Repeat after the commandsConfiguration is reloaded per my other comment.